lib/membrane_dashboard/charts/update.ex

```diff
@@ -23,16 +23,15 @@ defmodule Membrane.Dashboard.Charts.Update do
            |______________________________________________|
 
 
-  Steps needed for every chart to update data:
-  1. Query database to get all data from last 5 seconds.
-  2. Extract new paths (of pipeline elements) from the result of query (all paths that appeared for the first time in the last 5 seconds).
-  3. Create list of uPlot Series objects with `label` attribute (as maps: `%{label: path_name}`). One map for every new path.
-  4. Every path need to have value for every timestamp. Thus new series data for the time before update is filled with nils.
-  5. Extract new data (data for all paths for last 5 seconds; as list of lists) from database query result.
-  6. Truncate old data - delete its first 5 seconds (to maintain visibility of just last x minutes).
-  7. Concatenate truncated old data and new series data - it creates full data for the time before update.
-  8. Append new data to every path data.
-  9. Create map (of type `update_data_t`) serializable to ChartData in ChartsHook.
+  Firstly, queries database to get all data from last 5 seconds. Then applies following steps for every chart to update data:
```

Comment:
```suggestion
  Firstly, queries the database to get all the data for the last  5 seconds. Then applies the following steps to every chart to update the data:
```

Reply:
I don't want to correct everty seconds line here. Can you ran this whole instruction via `grammarly` or something like this?